apps/api/src/user/enum/user-organization-enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum UserOrganizationRoleEnum {
```

Comment:
@PhamAnhHoang , this should be in libs because this could be reused on api/platform/mobile, etc.